stream.go

```diff
@@ -107,13 +84,35 @@ type ClientStream interface {
 	// CloseSend closes the send direction of the stream. It closes the stream
 	// when non-nil error is met.
 	CloseSend() error
-	// Stream.SendMsg() may return a non-nil error when something wrong happens sending
-	// the request. The returned error indicates the status of this sending, not the final
-	// status of the RPC.
+	// Context returns the context for this stream.
```

Comment:
Sorry...can you add this, too (only for `ClientStream`):

`It should not be called until after Header or RecvMsg has returned.  Once called, subsequent client-side retries are disabled.`

Reply:
Done. Good note - that's unexpected (to me).